packages/react-native-bpk-component-flat-list/src/BpkRadioIcon.android.js

```diff
@@ -75,6 +67,7 @@ const BpkRadioIcon = ({ selected }: Props) => {
 
 BpkRadioIcon.propTypes = {
   selected: PropTypes.bool,
+  tintColor: PropTypes.string.isRequired,
```

Comment:
This is a breaking change strictly. Could we default it to `colorBlue500` to prevent that?

Reply:
This component is only used inside the component and isn't exported to consumers, so I don't think this is breaking?

Reply:
I just realised how ridiculous this sounds 😂 

> This component is only used inside the component

I mean it's only used by flatlist lol

Reply:
Right so we never export it at all?

Reply:
That's right - here's `index.js` for this package: https://github.com/Skyscanner/backpack-react-native/blob/master/packages/react-native-bpk-component-flat-list/index.js